src/domains/messages-app/10_function_messages_xl.tf

```diff
@@ -0,0 +1,598 @@
+
+  name                                      = format("%s-weu-com-citizen-func-snet-%d", local.product, count.index + 1)
+  address_prefixes                          = [var.cidr_subnet_appmessages_xl[count.index]]
+  resource_group_name                       = data.azurerm_virtual_network.vnet_common.resource_group_name
+  virtual_network_name                      = data.azurerm_virtual_network.vnet_common.name
+  private_endpoint_network_policies_enabled = false
+
+  service_endpoints = [
+    "Microsoft.Web",
+    "Microsoft.AzureCosmosDB",
+    "Microsoft.Storage",
+  ]
+
+  delegation = {
+    name = "default"
+    service_delegation = {
+      name    = "Microsoft.Web/serverFarms"
+      actions = ["Microsoft.Network/virtualNetworks/subnets/action"]
+    }
+  }
+}
+
+module "app_messages_function_xl" {
+  count  = var.app_messages_count
+  source = "github.com/pagopa/terraform-azurerm-v3//function_app?ref=v8.27.0"
+
+  resource_group_name = azurerm_resource_group.app_messages_rg[count.index].name
+  name                = format("%s-weu-com-citizen-func-%d", local.product, count.index + 1)
+  domain              = "MESSAGES"
+  location            = azurerm_resource_group.app_messages_rg[count.index].location
+
+  health_check_path            = "/api/v1/info"
+  health_check_maxpingfailures = 2
+
+  runtime_version                          = "~4"
+  node_version                             = "18"
+  always_on                                = true
+  application_insights_instrumentation_key = data.azurerm_application_insights.application_insights.instrumentation_key
+
+  app_service_plan_info = {
+    kind                         = "Linux"
+    sku_tier                     = "PremiumV3"
+    sku_size                     = "P2mv3"
+    maximum_elastic_worker_count = 0
+    worker_count                 = null
+    zone_balancing_enabled       = false
+  }
+
+  storage_account_info = {
+    account_kind                      = "StorageV2"
+    account_tier                      = "Standard"
+    account_replication_type          = "GZRS"
+    access_tier                       = "Hot"
+    advanced_threat_protection_enable = true
+    use_legacy_defender_version       = true
+    public_network_access_enabled     = false
+  }
+
+  app_settings = merge(
+    local.function_app_messages.app_settings_common,
+    {
+      FUNCTIONS_WORKER_PROCESS_COUNT = 8
+    }
+  )
+
+  subnet_id = module.app_messages_snet[count.index].id
+
+  allowed_subnets = [
+    module.app_messages_snet[count.index].id,
+    data.azurerm_subnet.app_backendl1_snet.id,
+    data.azurerm_subnet.app_backendl2_snet.id,
+    data.azurerm_subnet.apim_snet.id,
+  ]
+
+  allowed_ips = concat(
+    [],
+    local.app_insights_ips_west_europe,
+  )
+
+  # Action groups for alerts
+  action = [
+    {
+      action_group_id    = data.azurerm_monitor_action_group.error_action_group.id
+      webhook_properties = {}
+    }
+  ]
+
+  client_certificate_mode = "Required"
```

Comment:
Sure? I'd remove this one

Reply:
I just copied the old file 😢 are you sure that we can safely remove this property?

Reply:
Yep